app/assets/javascripts/discourse/app/lib/utilities.js

```diff
@@ -122,12 +122,6 @@ export function selectedText() {
         ? range.commonAncestorContainer
         : range.commonAncestorContainer.parentElement;
 
-    // ensure we never quote text in the post menu area
-    const postMenuArea = ancestor.querySelector(".post-menu-area");
-    if (postMenuArea) {
-      range.setEndBefore(postMenuArea);
-    }
-
```

Comment:
This was a huge source of jankiness as it means selection would change without user interaction